app/src/main/java/wishcantw/vocabulazy/audio/VLTextToSpeech.java

```diff
@@ -128,6 +135,14 @@ boolean isLanguageAvailable() {
         return (mTextToSpeech.isLanguageAvailable(Locale.ENGLISH) >= 0 && mTextToSpeech.isLanguageAvailable(Locale.TAIWAN) >= 0);
     }
 
+    public void mute() {
+        AppPreference.getInstance().setPlayerVolume(0.0f);
+    }
+
+    public void unMute() {
+        AppPreference.getInstance().setPlayerVolume(1.0f);
+    }
+
```

Comment:
@allencheng07 
I am curious about 2 things.
1. the input value of function setPlayerVolume() is based on the mobile phone setting or our apps??
2. mute() and unMute() seems two new functions after this commit, but I don't see anywhere to use them? Are these added for future usage?